internal/arr/config/arr.go

```diff
@@ -23,6 +23,8 @@ func RegisterArrFlags(flags *flag.FlagSet) {
 	flags.Bool("form-auth", false, "Use form based authentication")
 	flags.Bool("enable-unknown-queue-items", false, "Enable unknown queue items")
 	flags.Bool("enable-additional-metrics", false, "Enable additional metrics")
+  flags.Bool("skip-history", false, "Skip History metrics query")
+  flags.Bool("skip-missing", false, "Skip Wanted/Missing metrics query")
```

Comment:
Tabs

Reply:
I've never used go and I'm a "spaces not tabs" person so I was unaware of the standard or that I had done it differently than yours. I ran it through gofmt, given that's the official tool I'd imagine that's your expectation. Let me know if it needs anything else.

Reply:
I am also a spaces person as well, but Go has some opinions on formatting that I like to follow.

Reply:
Wholeheartedly agree -- if the language has a suggested standard, use it.